plugins/polio/api.py

```diff
@@ -246,31 +246,50 @@ def get_year(current_date):
             return today.year
 
     @staticmethod
```

Comment:
I don't think this method should be static (we can discuss in 1-1 if you want!)

Reply:
Fix in : 
- https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R226
- https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R226